README.rst

```diff
@@ -25,9 +25,10 @@ Install
 -------
 
 1. Install ``python3`` (on OSX you can run ``brew install python3``)
+#. Install ``virtualenv`` (on OSX you can run ``pip3 install virtualenv``)
 #. Initialize the python virtual environment:
 
-   a. ``virtualenv ./venv/ --python=python3.5``
```

Comment:
It's probably better to keep the option flag but generalize to `--python=python3`. Otherwise someone could end up with a python2 venv.

Reply:
Oh that's true, `python3 -m` would be better than `virtualenv`, but it's `python3 -m virtualenv env`, right? [link](https://packaging.python.org/guides/installing-using-pip-and-virtualenv/#creating-a-virtualenv).

Reply:
`python3 -m venv $env_name`

Reply:
`venv` is the `$env_name` though. And the python `-m`odule is virutalenv... right?

Reply:
@pumazi Ok, I've decided to not change this line anymore because I was using the latest version of Python 3 (3.7) and two tests were failing... on master branch.